### PR TITLE
Handled committee page blocks per authorization

### PR DESCRIPTION
**Pull request by HeahDude**
Closes [EM-32](http://jira.sensiolabs.net/browse/EM-32).

### CODE REVIEW

**Comment by HeahDude** [comment]
Until #174 is merged, review only last commit 666bd9c.

File: app/Resources/views/committee/show.html.twig [hunk 1]
```diff
@@ -5,10 +5,105 @@
...
+        </div>
+        <div class="committee-details">
+            <h5>Localisation</h5>
+            {#
+                TODO add static image
+            #}
+        </div>
+        <div class="committee-details">
+            <h5>Partager</h5>
+            {% if committee.facebookPageUrl %}
```
Comment (hhamon) [inline app/Resources/views/committee/show.html.twig]: Would be great to simplify to something like:

```twig
{% for network, link in committee.socialNetworksLinks %}
<a href="{{ link.url }}" target="_blank" class="committee-{{ network }}">
    {{ link.label }}
</a>
{% endfor %}
```

File: src/AppBundle/Twig/CommitteeExtension.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        return $this->authorizationChecker->isGranted(CommitteePermissions::CREATE, $committee);
+    }
+
+    public function canSee(Committee $committee): bool
+    {
+        return $this->authorizationChecker->isGranted(CommitteePermissions::SHOW, $committee);
+    }
+
+    public function getPath(string $routeName, Committee $committee): bool
```
Comment (hhamon) [inline src/AppBundle/Twig/CommitteeExtension.php]: Return type must be `string`.

File: src/AppBundle/Twig/CommitteeExtension.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        return $this->urlGenerator->generate($routeName, [
+            'uuid' => $committee->getUuid()->toString(),
+            'slug' => $committee->getSlug(),
+        ]);
+    }
+
+    public function getUrl(string $routeName, Committee $committee): bool
```
Comment (hhamon) [inline src/AppBundle/Twig/CommitteeExtension.php]: Return type must be `string`.

File: src/AppBundle/Twig/CommitteeExtension.php (hunk 2 above)
Comment (HeahDude) [inline src/AppBundle/Twig/CommitteeExtension.php]: Fixed.

File: src/AppBundle/Twig/CommitteeExtension.php (hunk 3 above)
Comment (HeahDude) [inline src/AppBundle/Twig/CommitteeExtension.php]: Fixed.

**Pull request merged**